src/auto_docking/scripts/auto_docking_client.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import roslib; roslib.load_manifest('kobuki_auto_docking')
+import rospy
+
+import actionlib
+from kobuki_msgs.msg import AutoDockingAction, AutoDockingGoal
+from actionlib_msgs.msg import GoalStatus
+from diagnostic_msgs.msg import DiagnosticArray
+
+
+class AutoDocking:
+    BATTERY_THRESHOLD = 4
+
+    def __init__(self):
+        self._ros_node = rospy.init_node('dock_drive_client_py', anonymous=True)
+        self._client = actionlib.SimpleActionClient('dock_drive_action', AutoDockingAction)
+        self._diagnostic_agg_sub = rospy.Subscriber("/diagnostics_agg", DiagnosticArray, self._listen_batteries)
+        self._go_docking = False
+
+    def _doneCb(self, status, result):
+        if status == GoalStatus.PENDING: state='PENDING'
+        elif status == GoalStatus.ACTIVE: state='ACTIVE'
+        elif status == GoalStatus.PREEMPTED: state='PREEMPTED'
+        elif status == GoalStatus.SUCCEEDED: state='SUCCEEDED'
+        elif status == GoalStatus.ABORTED: state='ABORTED'
+        elif status == GoalStatus.REJECTED: state='REJECTED'
+        elif status == GoalStatus.PREEMPTING: state='PREEMPTING'
+        elif status == GoalStatus.RECALLING: state='RECALLING'
+        elif status == GoalStatus.RECALLED: state='RECALLED'
+        elif status == GoalStatus.LOST: state='LOST'
+        # Print state of action server
+        print 'Result - [ActionServer: ' + state + ']: ' + result.text
+        self._go_docking = False
+
+    def _feedbackCb(self, feedback):
```

Comment:
Do we need the feedback? Otherwise remove.

Reply:
just for debugging. I would keep it until we are 100% sure that everything is working as expected.